portfolio/src/main/java/com/google/sps/servlets/Astro.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+
+@WebServlet("/zodiacsigns")
+public class Astro extends HttpServlet {
+  private Map<String, Integer> zodiacSign = new HashMap<>();
```

Comment:
Are you considering writing to datastore instead in the future? Or are you fine with having the vote counts reset periodically?